libs/block-content/website/src/lib/tik-tok-video-block.tsx

```diff
@@ -7,6 +7,27 @@ export const isTikTokVideoBlock = (block: Block): block is TikTokVideoBlockType
 
 export const TikTokVideoBlockWrapper = styled('div')``
 
+export const TikTokVideoBlockPlayer = styled('div')`
```

Comment:
You can use react-player for this. See youtube or vimeo

Reply:
I used iframe directly, as react-player wouldn't run (some cross-origin issues)

Reply:
My bad, didn't see Tiktok was not supported by react-player as it TikTok doesn't have a non iframe API.

Can we atleast use a CSS specifity of 1, not hardcode widths/heights and give it a proper title?